### ISSUE TITLE
Training setting

### ISSUE THREAD

**Issue by Steve-Tod** [yifita/deep_cage#8]
Hi, thank you for making the code public. It seems that you provided the [training code](https://github.com/yifita/deep_cage/blob/9ede51d22cf11740ead5c93ccc293ca46f68dfd4/cage_deformer_3d.py#L156) but didn't provide the default training options. 

For example, in the paper, there are losses like symmetry loss but the default weight for the loss in `option.py` is 0.

Could you please the default arguments you used for training?

**Comment by yifita** [comment]
Hi @Steve-Tod , i'm working on publishing the training code soon. The one you found is not for SURREAL data.

**Comment by yifita** [comment]
Hi Steve, I uploaded the instructions to train SURREAL data. Feel free to test out and let me know if you have questions.
